### DISCUSSION TITLE
App started but doesn't appear to be doing anything

### DISCUSSION THREAD

**Discussion by xander710** [MRColorR/money4band#46] [Q&A] [answered]
I have followed the installation process for windows
- Ensured that virtualization is enabled
- Installed docker
- Ran the automated setup for all app login details
- Started the app using the powershell window

After doing all this, a docker container starts running but nothing is appearing in the dashboards.
I am also unable to claim the earnapp node.

Anyone know what the problem could be?

**Comment by MRColorR** [comment, marked as answer]
Hi, The windows powershell is under rework to enable new functionalities.
The updated script will be released with the next patch.

Il Sab 20 Mag 2023, 11:26 xander710 ***@***.***> ha scritto:

**Comment by MRColorR** [comment]
Thanks for your patience :) The first (partial)patch for Windows Autoconfiguration has been pushed to the dev branch and the complete patch should be released to the public at the end of the week, so expect a new release during the weekend:)

for now you can proceed with the manual configuration if you wish :)
